src/webui/components/AutoComplete/styles.js

```diff
@@ -50,3 +51,10 @@ export const InputField = ({ color, ...others }: IInputField) => (
     }}
   />
 );
+
+export const SuggestionContainer = styled(Paper)`
+  && {
+    max-height: 500px;
+    overflow-y: scroll;
```

Comment:
Should be probably `auto` as otherwise you always have a scrollbar in some browsers.